src/browser_action/components/BrowserActionApp.jsx

```diff
@@ -49,6 +51,7 @@ export default class BrowserActionApp extends React.Component {
     super(props);
     this.state = {
       extractedProduct: props.extractedProduct,
+      showStudyInvitation: false,
```

Comment:
I think this is fine for this PR, but if we add any other "views" like this we should look into writing some sort of router or something that manages which view is "active". Otherwise figuring out which pages is active and the flow through pages will involve some complex if/then logic and several booleans like this.

Reply:
Can you share an example of what that might look like?

Reply:
Could be as simple as:

```js
class BrowserActionApp {
  constructor(props) {
    super(props);
    this.state = {
      currentPage: 'productListing',
    };
  }

  render() {
    switch (this.state.currentPage) {
      case 'productListing':
        return <TrackedProductList />;
      case 'studyEnrollment':
        return <StudyEnrollment />;
    }
  }
}
```

Or something as complex as [React Router](https://reacttraining.com/react-router/web/guides/quick-start), depending on our needs.